### PR TITLE
Make the WebPage being able to compile and render on brower

### PR DESCRIPTION
**Pull request by kaizhangNV**
By this PR, our demo page can compile a slang compute shader code, and render the result to the screen.
Specifically,

We setup two pipelines by using WebGPU APIs, one is a compute pipeline doing the computation and write the results to a output storage texture.

Another one is a pass through graphics pipeline which just output the storage texture to the screen.

We also polish the website a little bit to make it looks not like a draft doodle.

### CODE REVIEW

**Comment by kaizhangNV** [comment]
image

**Comment by kaizhangNV** [comment]
The page will looks like this.

File: index.html [hunk 1]
```diff
@@ -1,31 +1,71 @@
 <html>
-  <script src="try-slang.js"> </script>
+<style>
+textarea {
```
Comment (csyonghe) [inline index.html:3]: As a cleanup, we should put the css file in a separate location and reference it from the html.

File: index.html [hunk 2]
```diff
@@ -1,31 +1,71 @@
...
 
-void writeArray(inout float3 a[4])
-{
-    a[0] = float3(1, 1, 1);
-    a[1] = float3(1, 1, 1);
-    a[2] = float3(1, 1, 1);
-    a[3] = float3(1, 1, 1);
-}
+      <div class="w3-row-padding">
+        <div id="WebGPU-status-bar" style="width: 100%; height: 50px;"></div>
```
Comment (csyonghe) [inline index.html:40]: we should avoid doing inline styles like this, instead put the style in the css file as:
```
#WebGPU-status-bar {
    width: 100%;
    height: 50px;
}
```

Also consider using the `flex` layout to make the page resizable.

File: build.mk [hunk 3]
```diff
@@ -7,6 +7,20 @@ endif
 
 endef
 
+OS_NAME :=
```
Comment (csyonghe) [inline build.mk:10]: What does the build process do?

**Review by csyonghe — APPROVED** [review]

**Pull request merged**